roles/confluent.variables/defaults/main.yml

```diff
@@ -233,12 +237,16 @@ kafka_broker_inter_broker_listener_name: "{{ 'broker' if kafka_broker_configure_
 # Uses custom filter to create a list of all sasl_protocols, removes ['none'], and reduces to unique items
 kafka_broker_sasl_enabled_mechanisms: "{{ kafka_broker_listeners | get_sasl_mechanisms(sasl_protocol) | difference(['none']) | unique }}"
 
-### Only use to customize Linux User Kafka Service runs with. User must exist on host.
+### Only use to customize Linux User Kafka Service runs with. Default user is cp-kafka
```

Comment:
Modify this variable to customize the Linux User that the Kafka Broker Service runs with. Default user is cp-kafka.